### PR TITLE
Jhony/install supported xblocks

### PR DESCRIPTION
**Pull request by jfavellar90**

### CODE REVIEW

**Comment by jfavellar90** [comment]
@felipemontoya @diegomillan

File: requirements/edx/github.txt [hunk 1]
```diff
@@ -83,7 +83,7 @@ git+https://github.com/edx/edx-val.git@0.0.9#egg=edxval==0.0.9
 git+https://github.com/pmitros/RecommenderXBlock.git@v1.1#egg=recommender-xblock==1.1
 git+https://github.com/solashirai/crowdsourcehinter.git@518605f0a95190949fe77bd39158450639e2e1dc#egg=crowdsourcehinter-xblock==0.1
 -e git+https://github.com/pmitros/RateXBlock.git@367e19c0f6eac8a5f002fd0f1559555f8e74bfff#egg=rate-xblock
--e git+https://github.com/pmitros/DoneXBlock.git@857bf365f19c904d7e48364428f6b93ff153fabd#egg=done-xblock
```
Comment (jfavellar90) [inline requirements/edx/github.txt]: Este xblock como ya estaba en los github requirements de openedx, simplemente lo actualice a la version de upstream

**Comment by jfavellar90** [comment]
Tests ok @felipemontoya @diegomillan

**Comment by felipemontoya** [comment]
I think this is fine now. How about squashing it? :+1:

**Comment by jfavellar90** [comment]
Will squash and then merge for deploy

**Pull request merged**